addons/packages/pinniped/0.4.4/bundle/config/overlay/update_overlay.yaml

```diff
@@ -0,0 +1,31 @@
+#@ load("@ytt:overlay", "overlay")
```

Comment:
this overlay seems like it is yet another duplicated piece of logic throughout multiple packages. do we have plans to get rid of this duplication? i remember talking about this with @danniel1205 a while back.